### PR TITLE
Fix the download link on kurl.sh

### PR DESCRIPTION
**Pull request by jala-dx**
Fix the HEAD request to match the Request/URL origin.

### CODE REVIEW

File: src/components/DownloadAirgapBundle.js [hunk 1]
```diff
@@ -82,13 +85,7 @@ cat install.sh | sudo bash -s airgap
...
                   <span className="u-fontSize--small u-fontWeight--normal u-color--scorpion u-lineHeight--normal u-marginBottom--more"> You can get the airgap bundle a couple of ways. You can download it directly from here or via a CLI. </span>
                   <div className="u-marginTop--normal">
-                    <button
-                      type="button"
-                      className="Button primary"
-                      onClick={() => this.handleDownloadBundle()}
-                    >
-                      Download airgap bundle
-                    </button>
+                  <a href={downloadUrl}> Download airgap Bundle </a>
```
Comment (jala-dx) [inline src/components/DownloadAirgapBundle.js:88]: style is changed here though.

File: src/components/DownloadAirgapBundle.js [hunk 2]
```diff
@@ -82,13 +85,7 @@ cat install.sh | sudo bash -s airgap
                   <div className="FormLabel u-marginBottom--5"> Download airgap installer </div>
                   <span className="u-fontSize--small u-fontWeight--normal u-color--scorpion u-lineHeight--normal u-marginBottom--more"> You can get the airgap bundle a couple of ways. You can download it directly from here or via a CLI. </span>
                   <div className="u-marginTop--normal">
-                    <button
-                      type="button"
-                      className="Button primary"
-                      onClick={() => this.handleDownloadBundle()}
```
Comment (emosbaugh) [inline src/components/DownloadAirgapBundle.js:88]: is handleDownloadBundle still used? if not can you remove dead code?

File: src/components/DownloadAirgapBundle.js (hunk 1 above)
Comment (emosbaugh) [reply src/components/DownloadAirgapBundle.js:88]: What is the name of the file that is downloaded?

Comment (jala-dx) [reply src/components/DownloadAirgapBundle.js:88]: It is either `later.tar.gz` or `<sha>.tar.gz`

Screen Shot 2022-06-28 at 8 43 55 AM

Comment (emosbaugh) [reply src/components/DownloadAirgapBundle.js:88]: Great

File: .env.development [hunk 3]
```diff
@@ -1,5 +1,5 @@
 KURL_INSTALLER_URL="https://staging.kurl.sh/installer"
-KURL_BUNDLE_URL="https://k8s.staging.kurl.sh/bundle"
+KURL_BUNDLE_URL="https://staging.kurl.sh/bundle"
```
Comment (jala-dx) [inline .env.development:2]: this is basically reverting the previous PR

File: .env.staging [hunk 4]
```diff
@@ -1,5 +1,5 @@
 KURL_INSTALLER_URL="https://staging.kurl.sh/installer"
-KURL_BUNDLE_URL="https://k8s.staging.kurl.sh/bundle"
+KURL_BUNDLE_URL="https://staging.kurl.sh/bundle"
```
Comment (jala-dx) [inline .env.staging:2]: this is basically reverting the previous PR

File: src/components/App.js [hunk 5]
```diff
@@ -116,9 +105,10 @@ class AppComponent extends React.Component {
     const { loadingBundleUrl, installerData, selectedSpec, fetchingInstallerDataError } = this.state;
     const { isMobile } = this.props;
     const sha = this.props.sha;
-    const bundleUrl = `curl ${process.env.API_URL}/${sha} | sudo bash`
+    const bundleUrl = `curl -LO ${process.env.KURL_BUNDLE_URL}/${sha} | sudo bash`
```
Comment (emosbaugh) [inline src/components/App.js:108]: i think bundleUrl is a bad name here for this variable. perhaps rename it to installerUrl and revert this change

```suggestion
    const installerUrl = `curl -LO ${process.env.API_URL}/${sha} | sudo bash`
```

**Review by emosbaugh — CHANGES_REQUESTED** [review]

File: src/components/App.js [hunk 6]
```diff
@@ -164,13 +154,7 @@ cat install.sh | sudo bash -s airgap
...
                 </div>
                 <div className="u-marginTop--normal u-marginBottom--normal">
-                  <button
-                    type="button"
-                    className="Button secondary"
-                    onClick={() => this.handleDownloadBundle()}
-                  >
-                    Download airgap bundle
-                  </button>
+                <a href={downloadUrl}> Download airgap Bundle </a>
```
Comment (emosbaugh) [inline src/components/App.js:157]: if you wrap the button in an a tag will it fix the styling?

```
                    <a href={downloadUrl}>
                      <button
                        type="button"
                        className="Button primary"
                      >
                      Download airgap bundle
                      </button>
                    </a>
```

File: src/components/DownloadAirgapBundle.js [hunk 7]
```diff
@@ -18,7 +18,9 @@ class DownloadAirgapBundle extends React.Component {
   }
 
   checkS3Response = async (sha) => {
-    const url = `${process.env.KURL_BUNDLE_URL}/${sha}.tar.gz`
+    const bundleUrl = `${process.env.KURL_INSTALLER_URL}/${sha}.tar.gz`
```
Comment (emosbaugh) [inline src/components/DownloadAirgapBundle.js:21]:
```suggestion
    const bundleUrl = `${process.env.KURL_BUNDLE_URL}/${sha}.tar.gz`
```

File: src/components/App.js [hunk 8]
```diff
@@ -116,9 +105,10 @@ class AppComponent extends React.Component {
     const { loadingBundleUrl, installerData, selectedSpec, fetchingInstallerDataError } = this.state;
     const { isMobile } = this.props;
     const sha = this.props.sha;
-    const bundleUrl = `curl ${process.env.API_URL}/${sha} | sudo bash`
+    const bundleUrl = `curl -LO ${process.env.KURL_BUNDLE_URL}/${sha} | sudo bash`
+    const downloadUrl = `${process.env.KURL_BUNDLE_URL}/${sha} | sudo bash`
```
Comment (emosbaugh) [inline src/components/App.js:109]:
```suggestion
    const downloadUrl = `${process.env.KURL_BUNDLE_URL}/${sha}.tar.gz`
```

File: src/components/DownloadAirgapBundle.js [hunk 9]
```diff
@@ -82,13 +73,7 @@ cat install.sh | sudo bash -s airgap
...
                   <span className="u-fontSize--small u-fontWeight--normal u-color--scorpion u-lineHeight--normal u-marginBottom--more"> You can get the airgap bundle a couple of ways. You can download it directly from here or via a CLI. </span>
                   <div className="u-marginTop--normal">
-                    <button
-                      type="button"
-                      className="Button primary"
-                      onClick={() => this.handleDownloadBundle()}
-                    >
-                      Download airgap bundle
-                    </button>
+                  <a href={downloadUrl}> Download airgap Bundle </a>
```
[the remark this answers is not in the document]
Comment (erquhart) [reply src/components/DownloadAirgapBundle.js:76]: That could cause issues - try adding the `Button` and `primary` classes to the link - styling should just work.

File: .env.development (hunk 3 above)
Comment (jala-dx) [reply .env.development:2]: Got it, let the requests go to [k8s](https://k8s.kurl.sh/bundle)  instead of [Cloudflare]( https://kurl.sh/bundle), writing it here for my own memory)

File: .env.production [hunk 10]
```diff
@@ -1,5 +1,5 @@
 KURL_INSTALLER_URL="https://kurl.sh/installer"
-KURL_BUNDLE_URL="https://k8s.kurl.sh/bundle"
+KURL_BUNDLE_URL="https://kurl.sh/bundle"
```
[the remark this answers is not in the document]
Comment (jala-dx) [reply .env.production:2]: Got it, let the requests go to k8s instead of Cloudflare, writing it here for my own memory)

File: src/components/App.js (hunk 5 above)
Comment (erquhart) [reply src/components/App.js]: @jala-dx this wasn't resolved

**Review by erquhart — CHANGES_REQUESTED** [review]

File: src/components/App.js [hunk 11]
```diff
@@ -156,21 +146,22 @@ cat install.sh | sudo bash -s airgap
...
                 </div>
                 <div className="u-marginTop--normal u-marginBottom--normal">
-                  <button
-                    type="button"
-                    className="Button secondary"
-                    onClick={() => this.handleDownloadBundle()}
-                  >
-                    Download airgap bundle
+                <a href={downloadUrl}> 
+                  <button
```
[the remark this answers is not in the document]
Comment (jala-dx) [reply src/components/App.js]: @erquhart - Does it looks valid now ? (the previous syntax worked as well fyi)

**Review by emosbaugh — APPROVED** [review]

**Review by erquhart — APPROVED** [review]
